scripts/local-generate-svg.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+export SOURCECRED_DIRECTORY=$(pwd)/sourcecred_data
+SOURCECRED_GITHUB_TOKEN=`cat secrets/token`
+REPOS=`cat repositories.txt`
+
+cd mkweights
+yarn
+cd ..
```

Comment:
Hmm, so I guess my worry here is that there are quite a few dependencies. For example, not everyone has yarn. and node is a beast. Is this generated on a user computer, locally? I'm thinking either we can provide a Dockerfile (and container?) to issue commands with (and the one dependency is docker of course) or the generation can be done in CI that has a recipe to install everything. Thoughts?

Reply:
> Hmm, so I guess my worry here is that there are quite a few dependencies.

That is definitely true. I'm involved with a couple of discussions to make depending on sourcecred simpler. For example https://github.com/sourcecred/widgets/issues/8

>  For example, not everyone has yarn. and node is a beast.

With regards to yarn. Sourcecred has a yarn lockfile committed but not the npm `package-lock.json` counterpart. Thus your dependencies would be yarn + npm + node if I had kept the npm lock variant. That's why I switch to yarn here.

> Is this generated on a user computer, locally?

As `local-*` is intended for debugging it's useful to have node installed. And yarn is pretty stable so `npm install -g yarn` would be sufficient if you do have node and npm.

The official `node` docker container has yarn installed. In [`.drone.yml`](https://github.com/sfosc/sourcecred/blob/31716d69efb74384bd37715cea4919fcf08c7bb8/.drone.yml#L16) I use `node:10` as that's the active LTS node version. Should you prefer avoiding node you might try using `node:10` or `node:12` (current stable and tested by sourcecred test suite) and including the repository root as a bind volume. Personally I've not tested the local-* with this yet. But it's essentially what our Drone build does.

> I'm thinking either we can provide a Dockerfile (and container?) to issue commands with (and the one dependency is docker of course) or the generation can be done in CI that has a recipe to install everything. Thoughts?

That is what this PR will do. The `.drone.yml` file configures the job run on a daily interval. That calls `scripts/rebuild-site.sh` with `node:10` as the main docker image to do so.

Reply:
If local is for debugging, and there is a configuration with Drone to run programatically otherwise, I'm good here. What we don't want is an entirely un-reproducible thing.

Reply:
Yes the main script used for CI is scripts/rebuild-site.sh
Which is also usable locally.